React/apis-post.md

```diff
@@ -60,16 +72,22 @@ Now observe what happens when we submit the form in our application. Check in wi
 - How can you handle validation errors in a real API, like we made before?
 - How could you delete pets from the API with your React App?
   - With your neighbor implement this functionality.
-  - Verify that the pets dissapear from the json file.
 
-When you are ready you can see [our solution](examples/deletePet.js).
+## Exercise
+
+At this point you can list students and add new students, but you cannot delete them.  For this exercise take the working code from this [point in the textbook](https://github.com/AdaGold/ada-students/tree/12-api-post), and add a `delete` button to each student and give the app the ability to delete students.  This should cause a `DELETE` request to be made from the API.  
```

Comment:
```suggestion
At this point you can list students and add new students, but you cannot delete them.  For this exercise take the working code from this [point in the textbook](https://github.com/AdaGold/ada-students/tree/12-api-post), and add a `delete` button to each student and give the app the ability to delete students.  Every time a user clicks on a "delete" button for a specific student, our app should send a `DELETE` request to the API.  
```

The phrase "This should cause a `DELETE` request to be made from the API" should be changed to "This should send a `DELETE` request to the API". Optionally, I've suggested changing the first part "This" to "every time a user clicks on a delete button"

Reply:
They will have used a `DELETE` request in [Wave 3 of Ada Pets Node](https://github.com/AdaGold/ada-pets-node#wave-3-remove-pet).  

Though we maybe should have added more info there.